be/src/http/utils.cpp

```diff
@@ -164,14 +164,25 @@ void do_dir_response(const std::string& dir_path, HttpRequest* req) {
     }
 
     const std::string FILE_DELIMETER_IN_DIR_RESPONSE = "\n";
+    const std::string FILE_NAME_SIZE_DELIMETER = ",";
```

Comment:
I have a question. If letter `,` is included in a filename, can this work normally as expected?